eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -240,6 +240,8 @@ jobs:
       VCPKG_DEFAULT_TRIPLET: 'x64-windows-static'
     steps:
       - template: /eng/common/pipelines/templates/steps/check-spelling.yml
+        parameters:
+          ContinueOnError: false
```

Comment:
To me, "continue on error" reads as "exit once the first error was detected", i.e. giving user one spelling error at a time: you fix one, it fails on the next one. It probably would've been annoying :)